src/crate/theme/rtd/crate/sidebartoc.html

```diff
@@ -16,14 +16,19 @@
     </div>
   </li>
 
-  <!-- Home. -->
-  {% if project == 'CrateDB: Guide' and pagename == 'home/index' %}
-    <li class="navleft-item current"><a class="current-active" href="/docs/guide/home/">Overview</a></li>
+  <!-- Section A: -->
+  {% if project == 'CrateDB: Guide' %}
+    <!-- Show all TOC entries without the Guide heading -->
+    <li class="current toctree-l0">
+      {{ toctree(maxdepth=-1|toint, titles_only=True, collapse=True, includehidden=theme_globaltoc_includehidden|tobool) }}
+    </li>
   {% else %}
+    <!-- Show Guide sections as links in other projects -->
     <li class="navleft-item"><a href="/docs/guide/home/">Overview</a></li>
+    <li class="navleft-item"><a href="/docs/guide/start/">Getting Started</a></li>
+    <li class="navleft-item"><a href="/docs/guide/handbook/">Handbook</a></li>
```

Comment:
Maybe this can be adjusted to just point to `guide/` again. Does anything speak against it?

- https://github.com/crate/cratedb-guide/issues/381

Reply:
You are probably right in your assessment that this won't work. So we may find a different solution for mitigating the blank page?

Reply:
That's probably the dilemma? Or just re-add the main body content there?